### DISCUSSION TITLE
fits with missing data result in different estimated models

### DISCUSSION THREAD

**Discussion by andybeet** [James-Thorson-NOAA/dsem#18] [Show and tell]
From our discussion last week, i have provided a [link](https://github.com/andybeet/testingdsem?tab=readme-ov-file#results) to some results.
A comparison of the fitted dsem from the [vignette](https://james-thorson-noaa.github.io/dsem/articles/vignette.html#comparison-with-dynamic-linear-models) with the same model, where some, (25%), of the data have been randomly set to missing. It seems like there is some sensitivity to missing data.

**Comment by James-Thorson-NOAA** [comment]
Thanks for sharing!  I switched it to the Discussions tab.  I think it's a good demonstration of nonparametric bootstrap testing for missing data.

**Comment by andybeet** [comment]
The recent paper by [Fang & Wang (2024)](https://www.tandfonline.com/doi/full/10.1080/10705511.2023.2287967) is very informative. To summarize, there has been insufficient research to determine when you'd expect accurate and precise results with missing data (for DSEMs). The models they simulate from in their paper are relatively simple and they show results using different kinds of missing data structure (homogeneous and heterogeneous) and differnt levels of % missing. The value of 25% missing data (homogeneous), the amount i used to demonstate this issue, is toward the low end according to Fang & Wangs lit search of published work. Take home: I think we should probably perform simulation tests for our models when we have substantial amounts of missing data to make sure the estimates are robust. On another note, it might also be worth running the models through multiple DSEM packages, like `Mplus` or `gsem` (STATA) as a comparison.
